queries.js

```diff
@@ -66,6 +76,49 @@ function addProduct(ctx) {
   });
 }
 
+async function getOrdersServedByEmployee(ctx) {
+  const session = driver.session();
+  const { name } = ctx.params;
+  const statement = `MATCH (:Employee {name: '${name}'})-[:ENTERED]->(o:Order) RETURN o`
+  const result = await session.run(statement);
+  session.close();
+
+  const orders = [];
+  for (const element of result.records) {
+    const orderId = element.get(0).identity.low; //ORDER ID
+    const totalPrice = element.get(0).properties.price.low; //ORDER TOTAL PRICE
+    const product = await getProductsFromOrder(orderId);
```

Comment:
kod för att filtrera på datumen:

Först konverera orders createdAt till string:
// convert orders createdAt object to string
        let orderCreated = element.get("o").properties.createdAt.year.low + 
        "-" + element.get("o").properties.createdAt.month.low + "-" +
        element.get("o").properties.createdAt.day.low;

Sen lägg bara till order i ordersarray om:

if (orderCreated.localeCompare(data.qStartDate) != -1 && orderCreated.localeCompare(data.qEndDate) != 1) {
           orders.push(//Kod...)
          }